### PR TITLE
BPS Works

### PR DESCRIPTION
**Pull request by manthanand**
# Quality Assurance Checklist 
To make reviews more efficient, please make sure the software feature meets the following standards and check everything off that meets the quality check. Once everything has been checked, the assigned reviewers will begin the review process. _Edit this description to check off the list._

There are exceptions with all guidelines. As long as your decisions are justified, then you are good! Contact the reviewers or the leads about any exceptions. 

## Requirements 
- [x] Followed Coding Style Guide 
- [x] Code Build checks pass 
- [x] No merge conflicts 
- [x] Software feature has test associated with it. 
- [x] Software feature has documentation for it updated.
- [x] If applicable, does software feature have simulation test associated with it (.json & out.json file)
- [x] Test provides useful information and uses relevant data to accurately represent BPS 
    - NOTE: If test file already exists, use that one 
- [ ] 2 Members of the team have already reviewed my PR 
    - NOTE: Approver will not look at PR if 2 other members haven't reviewed first 
- [x] Did you Have Fun?

## Things to Consider 
- [ ] Even if the above are checked, is this the best way of writing my code? 
    - It's possible to write code that works, but are there ways to make code more efficient?

### CODE REVIEW

File: Apps/Src/Temperature.c [hunk 1]
```diff
@@ -196,6 +196,9 @@ int32_t milliVoltToCelsius(uint32_t milliVolt){
     if (milliVolt < sizeof(voltToTemp)/sizeof(voltToTemp[0])) {
         return voltToTemp[milliVolt];
     }
+    else if (milliVolt < 5200 && milliVolt > 4800) { //temperature sensor is disconneted for scrutineering
```
Comment (tiandahuang) [inline Apps/Src/Temperature.c:199]: spelling error

File: Apps/Src/Temperature.c [hunk 2]
```diff
@@ -299,6 +302,7 @@ SafetyStatus Temperature_CheckStatus(uint8_t isCharging){
             }
         }
     }
+    printf("\n\r");
```
Comment (tiandahuang) [inline Apps/Src/Temperature.c:305]: probably meant to take this out

File: Drivers/Src/LTC2315.c [hunk 3]
```diff
@@ -56,13 +56,13 @@ uint16_t LTC2315_Read() {
...
     uint8_t rxdata[2] = {0xff, 0xff};
 
-    LTC2315_wakeup_sleep();
-
     uint8_t count = 0;
     do {
     OSSchedLock(&err);
     assertOSError(err);
 
+    LTC2315_wakeup_sleep();
```
Comment (tiandahuang) [inline Drivers/Src/LTC2315.c:64]: should the wakeup sleep be inside the do while loop? 
idk if we want to move it back without testing tho

**Review by tiandahuang — APPROVED** [review]

File: Drivers/Src/LTC2315.c (hunk 3 above)
Comment (manthanand) [reply Drivers/Src/LTC2315.c:64]: i think it is fine where it is for now. ideally, the do while does not while too much

**Pull request merged**